wherehows-web/tests/acceptance/login-test.js

```diff
@@ -25,16 +38,73 @@ test('should render login form', function(assert) {
   });
 });
 
-test('should display error message with empty credentials', async function(assert) {
-  assert.expect(2);
+// test('should display error message with empty credentials', async function(assert) {
+//   assert.expect(2);
+//   await fillIn(loginUserInput, testUser);
+//   await click('button[type=submit]');
+//
+//   assert.ok(find('#login-error').text().length, 'error message element is rendered');
+//
+//   assert.equal(
+//     find('#login-error')
+//       .text()
+//       .trim(),
+//     invalidCredentials
+//   );
+// });
 
-  await click('button[type=submit]');
+test('Login with an valid password', function(assert) {
+  visit('/#/login');
```

Comment:
For tests, you don't need to use location based hash,although the app uses this, Testem prefers the default location type, so you can use that here i.e. `visit('/#/login')` should be `visit('/login')`. That should work fine.

This applies to similar usages below

Reply:
fixed